survey_sign_oca/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2025 Kencove - Mohamed Alkobrosli
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Survey Sign Oca",
+    "version": "16.0.1.0.0",
+    "category": "Surveys",
+    "website": "https://github.com/OCA/sign",
+    "author": "Kencove, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "depends": ["sign_oca", "survey", "survey_question_type_binary"],
```

Comment:
```suggestion
    "depends": ["sign_oca", "survey", "survey_question_type_binary"],
```
IMP the `survey_question_type_binary` dependency should not be necessary.

Reply:
it is important if the survey takers will give answers of type binary or multibinary or signature.

It is very important for them to sign, and I can not ignore this field as I need it, and it is available in an OCA module.

Reply:
but I will try to use get method if the field is present or not, I will try no problem.